src/Apps/Order2/Routes/Checkout/Components/ExpressCheckoutSection.tsx

```diff
@@ -33,16 +35,12 @@ export const ApplePay: React.FC<ApplePayProps> = ({ order }) => {
 }
 
 const orderFragment = graphql`
-  fragment ApplePay_order on CommerceOrder {
+  fragment ExpressCheckoutSection_order on Order {
```

Comment:
I know this PR is WIP but just to get ahead of theses things now and not need to do any refactoring, any relay component for a given `App` namespace needs to be prefixed with the order app name (without exception, due to a) the unique ID requirement; and b) sane logging / discovery at the query reporting level). 

So this component for example, idiomatically:

```tsx
const Order2ExpressCheckoutSection: React.FC<Order2ExpressCheckoutSection> = ({ order }) => {
  const data = useFragment(FRAGMENT, order)
  
  return (
    ...
  )
}

const FRAGMENT = graphql`
  fragment Order2ExpressCheckkoutSection_order on Order {
  
  }
`
```
(The above is the modern form verbatim)